op-batcher/batcher/channel_manager.go

```diff
@@ -34,26 +33,20 @@ type channelManager struct {
 	tip common.Hash
 
 	// Pending data returned by TxData waiting on Tx Confirmed/Failed
-
-	// pending channel builder
-	pendingChannel *channelBuilder
-	// Set of unconfirmed txID -> frame data. For tx resubmission
-	pendingTransactions map[txID]txData
-	// Set of confirmed txID -> inclusion block. For determining if the channel is timed out
-	confirmedTransactions map[txID]eth.BlockID
+	currentChannel *channel
```

Comment:
comments on the usage of currentChannel + channelQueue would be helpful.
As far as I can tell we only add to the currentChannel & then pull frames from any of the channels in the channelQueue (preferring older channels in the queue).